app-accessibility/piper/pipertts-1.0.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="Offline TTS with NCNN backend (Piper TTS)"
+DESCRIPTION="Offline TTS program using NCNN library With Natural Voice"
+HOMEPAGE="https://github.com/gyroing/PiperTTS-HaikuOS/"
+COPYRIGHT="2025 GYROING"
+LICENSE="BSD (3-clause)"
+ARCHITECTURES="x86_64"
+REVISION="1"
+RECIPE_API_VERSION="2"
+SOURCE_URI="https://github.com/gyroing/PiperTTS-HaikuOS/archive/refs/tags/piper.tar.gz"
+CHECKSUM_SHA256="b50256cde0232eb52895a124ce0620cc419fff78df203e0c0ad275e5a2d8b8d0"
+SOURCE_URI_2="https://github.com/gyroing/ncnn/archive/refs/tags/ncnn-haikuports.tar.gz"
```

Comment:
Any reason for using your own fork here, from a quick look I don't see any changes with upstream original sources/commits?
Maybe add this as a seperate package?